closure/goog/structs/heap_test.js

```diff
@@ -205,11 +205,39 @@ function testInsertPeek2() {
   var h = new goog.structs.Heap();
 
   h.insert(1, 'b');
-  assertEquals('peak, Should be "b"', h.peek(), 'b');
+  assertEquals('peek, Should be "b"', 'b', h.peek());
   h.insert(3, 'd');
-  assertEquals('peak, Should be "b"', h.peek(), 'b');
+  assertEquals('peek, Should be "b"', 'b', h.peek());
   h.insert(0, 'a');
-  assertEquals('peak, Should be "a"', h.peek(), 'a');
+  assertEquals('peek, Should be "a"', 'a', h.peek());
   h.insert(2, 'c');
-  assertEquals('peak, Should be "a"', h.peek(), 'a');
+  assertEquals('peek, Should be "a"', 'a', h.peek());
+}
+
+function testInsertAllPeek1() {
+  var h1 = new goog.structs.Heap();
+  var h2 = getHeap();
```

Comment:
This test is difficult to read and understand because test logic that is important to the correctness of its assertions is external to the actual method.  This is not your fault - getHeap was here when you started - but I would be a lot happier about it if we had, say, a makeHeap method that took pairs and inserted them in the right order:

  var h1 = makeHeap();
  var h2 = makeHeap([0, 'a'], [2, 'c'], [1, 'b']);

This would make it completely obvious from just the method (and the "obvious" implementation of makeHeap) that the test was correct.  Would you mind pulling this method out as well?